packages/nextjs/app/page.tsx

```diff
@@ -15,7 +16,7 @@ const Home: NextPage = () => {
         <div className="px-5">
           <h1 className="text-center">
             <span className="block text-2xl mb-2">Welcome to</span>
-            <span className="block text-4xl font-bold">Scaffold-ETH 2</span>
+            <span className="block text-4xl font-bold">{projectConfig.name}</span>
```

Comment:
This page is mostly rewritten for new apps so I think no need to use variable here